yarn.lock

```diff
@@ -4420,13 +4420,111 @@
   resolved "https://registry.yarnpkg.com/@open-draft/until/-/until-1.0.3.tgz#db9cc719191a62e7d9200f6e7bab21c5b848adca"
   integrity sha512-Aq58f5HiWdyDlFffbbSjAlv596h/cOnt2DO1w3DOC7OJ5EHs0hd/nycJfiu9RJbT6Yk6F1knnRRXNSpxoIVZ9Q==
 
-"@ovh-ux/ng-ovh-api-wrappers@^4.0.11", "@ovh-ux/ng-ovh-api-wrappers@^4.0.4":
+"@ovh-ux/component-rollup-config@^11.0.1":
```

Comment:
Following diff has been fixed via #6772.

Before merging this Pull Request, a rebase will be required.

Reply:
Rebasing on `develop` branch in order to remove the actual diff on `yarn.lock` file.

Reply:
Rebased successfully.